zigpy/device.py

```diff
@@ -131,3 +132,13 @@ def ieee(self):
 
     def __getitem__(self, key):
         return self.endpoints[key]
+
+
+async def broadcast(app, profile, cluster, src_ep, dst_ep, grpid, radius,
```

Comment:
I'm not sure I see the reason for this here. Couldn't things just call `app.broadcast()` directly?

Reply:
@rcloran let me know if it is better to call those directly and I'll update the PR. Once it goes through, we could add ZHA config panel and targeted joins to home assistant.